packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -252,6 +268,12 @@ function GridItem( { categoryId, item, ...props } ) {
 										: __( 'Duplicate' )
 								}
 							/>
+							{ item.type === USER_PATTERNS && (
+								<MenuItem onClick={ () => exportAsJSON() }>
```

Comment:
Question: Should we close the dropdown menu after it's pressed? I feel like that might be doing too much. I could use some accessibility feedback here 🤔 .

Reply:
I'm no a11y expert so I'll defer to others here. One point to note though is that the same dropdown is closed when we rename a pattern etc so it should probably be consistent one way or the other.